README.md

```diff
@@ -179,6 +179,88 @@ cpu: Apple M1 Pro
 BenchmarkUnwrap_D_32_WithSetup-10    	  301116	      3619 ns/op	    8608 B/op	      64 allocs/op
 ```
 
+### Command Line Usage
+
+tr31 is a tool for managing both TDES and AES-derived unique keys per transaction (TR-31) key management.
+
+### USAGE 
+    tr31 [-v] [-algorithm] [-e] [-d]
+
+### EXAMPLES
+    tr31 -v 
+      Print the version of tr31 (Example: v1.0.0) 
+    tr31 -e 
+      Encrypt a card data block using the TR-31 transaction key 
+    tr31 -d 
+      Decrypt a card data block using the TR-31 transaction key
+
+### FLAGS
+    -e_va string 
+      Vault address where the encryption key is stored 
+    -e_tk string 
+      Vault token for authentication 
+    -e_kp string 
+      Path to the encryption key in the vault 
+    -e_kn string 
+      Name of the encryption key in the vault 
+    -e_ek string 
+      Encryption key
```

Comment:
We don't use abbreviations like `va` on moov projects. Call it `-vault-address` or `-key-path`

Reply:
change flags 
-vault_address string 
      Vault address where the encryption/decryption key is stored 
    -vault_token string 
      Vault token for authentication 
    -key_path string 
      Path to the encryption/decryption key in the vault 
    -key_name string 
      Name of the encryption/decryption key in the vault 
    -enc_key string 
      Encryption key
    -key_block string 
      Wrapped key block for decryption